src/commands/cmd_stream.cc

```diff
@@ -742,8 +742,6 @@ class CommandXRead : public Commander,
       auto s = stream_db.Range(streams_[i], options, &result);
       if (!s.ok()) {
         conn_->Reply(redis::MultiLen(-1));
-        LOG(ERROR) << "ERR executing XRANGE for stream " << streams_[i] << " from " << ids_[i].ToString() << " to "
```

Comment:
i am not sure how to trigger it yet, but look at the code, look like it works in XREAD instead of XRANGE.
i belive it is also a debug log?

Reply:
@torwig Could you take a look at this line?

Reply:
@enjoy-binbin Internally, both `XREAD` and `XRANGE` call the `Stream::Range` function. 
I think we can respond with an error here (as we do in line 634) in case of an error:

```
if (!s.ok() && !s.IsNotFound()) {
    return {Status::RedisExecErr, s.ToString()};
} 
```

Reply:
thanks. I will update it soon